src/applications/check-in/day-of/pages/CheckIn/DisplayMultipleAppointments.jsx

```diff
@@ -62,8 +63,9 @@ const DisplayMultipleAppointments = props => {
           })}
         </ol>
         <p data-testid="update-text">
-          <strong>Latest update:</strong>{' '}
-          {format(new Date(), "MMMM d, yyyy 'at' h:mm aaaa")}
+          <Trans i18nKey="latest-update" values={{ date: new Date() }}>
```

Comment:
Why does this one require the component wrapper?

Reply:
It's needed when there are html tags in the translated string, c.f. https://react.i18next.com/latest/trans-component#usage-with-simple-html-elements-like-less-than-br-greater-than-and-others-v10.4.0